querybook/webapp/components/DataTableViewOverview/DataTableViewOverview.tsx

```diff
@@ -82,6 +82,20 @@ function useRefreshMetastore(table: IDataTable) {
     return [handleRefreshTable, isRefreshing] as const;
 }
 
+function createChannelLinkDOM(customProperties) {
+    const channelsKey = 'channels';
+    const channelsValue = customProperties[channelsKey]?.toString() ?? '';
+
+    return channelsValue === '' ? null : (
+        <KeyContentDisplay
+            key={channelsKey}
+            keyString={titleize(channelsKey, '_', ' ')}
+        >
+            {channelsValue}
+        </KeyContentDisplay>
+    );
+}
+
```

Comment:
this is duplicating a lot of code specially for channels, could you have something like:

```
/**
* any custom properties in this array will be shown on top following the order of the array
*/
const pinnedCustomProperties = ['channels'];
```
then when you go over customPropertiesDOM, make orderedProperties show at the beginning

Reply:
- Refactored KeyContentDisplay to helper 
- Added `pinnedCustomProperties` to displayed pinned custom properties at top and unpinned custom properties at bottom
- Updated test plan with screenshots